packages/framework/src/Component/Domain/Multidomain/EmptyMultiDomainEntityClassProvider.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Shopsys\FrameworkBundle\Component\Domain\Multidomain;
 
-class MultiDomainEntityClassProvider implements MultiDomainEntityClassProviderInterface
+class EmptyMultiDomainEntityClassProvider implements MultiDomainEntityClassProviderInterface
```

Comment:
I would name it rather `DefaultMultiDomainEntityClassProvider`, what do you say?

Reply:
Also it would be nice to add annotation explaining its existence :slightly_smiling_face: Something like "The class ensures proper functionality after the component is separated into standalone package independent of framework. See Shopsys\FrameworkBundle\Component\Domain\Multidomain\MultidomainEntityClassFinderFacade. Now, in framework, yet another implementation is used, see Shopsys\FrameworkBundle\Model\MultiDomainEntityClassProvider"

Reply:
I understand, but imho i wouldnt write it as "Now it works like this, and then it will work like this, because there will be needed action in future to check this one and rewrite it, I know that this is wierd, but i would describe it as it was already seperated.